### PR TITLE
Created works section

### PR DESCRIPTION
**Pull request by K-mone**
In this PR. I have completed the following tasks:
-Set up the linters for HTML and CSS
-Updated my README file
-Created the HTML file for works section and about me section
-Added the CSS file 
-Added the required photos
-Created the .gitignore file and added node modules to that file

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,98 +1,280 @@
...
+    gap: 24px;
+  }
+
+  .feature {
+    display: flex;
+    padding: 0 24px;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+  }
```
Comment (Rhaegar121) [inline style.css:97]: Remove `padding: 0 24px;` from the feature div to fix the white space problem as shown in the screenshot.

Screenshot (178)

**Review by Rhaegar121 — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @K-mone  👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close lady

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: style.css (hunk 1 above)
Comment (Rhaegar121) [inline style.css]: Kindly remove `padding: 0 24px;` from the feature div to fix the white space problem as shown in screenshot.

Screenshot (178)

File: style.css [hunk 2]
```diff
@@ -1,98 +1,280 @@
...
 
-  nav a:hover {
-    color: var(--branding-color);
+  .card-wrapper {
+    display: flex;
+    padding: 0 24px;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
   }
```
Comment (Rhaegar121) [inline style.css]: Kindly remove `padding: 0 24px;` from the card-wrapper div too to fix the above problem.

**Comment by Rhaegar121** [comment]
Please kindly update your design to look exactly like the provided figma template.
`Your website should look exactly like the design (e.g. font, colors, images, text, margins) using the templates in Figma`

You can change your figma to developer mode in order to see the css code.

Screenshot (170)

**Comment by Rhaegar121** [comment]
I think you forgot about one requirement to add button interaction. **Kindly add this interaction to all your buttons.**

`Implement the button interactions (enable, hover, etc.)`

**Comment by Rhaegar121** [comment]
One last thing is to add the responsive design with `(min-width: 768px)` in your CSS. And implement the design according to the figma template as shown in the screenshot.

```
The website should be responsive, following these breakpoints:
Minimum mobile screen size should be 375px
Minimum desktop screen size should be 768px
Use responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels
```
Screenshot (179)

File: index.html [hunk 3]
```diff
@@ -63,9 +57,8 @@ <h3>I'm a Software Developer</h3>
         my work and experience! If you like what you see and have a project you
         need coded, don't hesitate to contact me.
       </p>
-
       <footer>
-        <div class="social-links d-flex justify-content-center">
+        <div class="social-links">
```
Comment (DanielAD10) [inline index.html:61]: I think you need to align the icon to the left as shown in figma.
And the gap between text and icon is too far. Please make sure the gap between text and icon to be the same.

**Review by DanielAD10 — CHANGES_REQUESTED** [review]
### STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️
---
Hi @K-mone 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights 💯
---
No Linter errors ✔️
Descriptive Pull request summary✔️
Good commit messages ✔️
Professional Readme file ✔️

### Required Changes ♻️
---
Check the comments under the review.

_Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers, and Happy coding!**👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

**Comment by DanielAD10** [comment]
I think you need to align the icon to the left as shown in figma.
And the gap between text and icon is too far. Please make sure the gap between text and icon to be the same.

Your website should look exactly like the design (e.g. font, colors, images, text, margins) using the templates in Figma.

ss1     fig1

**Review by DanielAD10 — CHANGES_REQUESTED** [review]
I think you need to solve first the Branch Conflict.

**Review by DanielAD10 — CHANGES_REQUESTED** [review]
###  STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @K-mone  👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close lady

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by DanielAD10** [comment]
I think you need to fix the alignment in Multi Post Stories. Please make sure the HTML, CSS, JavaScript Button and See More Button are left align as in Figma.
Screenshot (300)

**Comment by DanielAD10** [comment]
In your website, the color of HTML, Bootstrap, Ruby Button are not the same as Figma. And the See Project Button is not the same. Please make sure your website design layout is the same as Figma. 

This is from Figma 
Screenshot (301)

This is your design. 
Screenshot (305)

**Comment by DanielAD10** [comment]
I think you need to add the line under the vector as in Figma.

Screenshot (302)

**Comment by DanielAD10** [comment]
I would like to tell you to make sure font size, font design and icon are the same as in Figma.


This is Figma Design.
Screenshot (303)

This is your website Design.
Screenshot (304)

**Review by DanielAD10 — APPROVED** [review]
###  STATUS: APPROVED 🚦 ⚠️ ♻️ 

Hi @K-mone  👋, 

Good job so far!

## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**